security/cmd/node_agent/main.go

```diff
@@ -51,39 +51,39 @@ func init() {
 
 	flags := rootCmd.Flags()
 
-	flags.StringVar(&naConfig.ServiceIdentityOrg, "org", "", "Organization for the cert")
-	flags.DurationVar(&naConfig.WorkloadCertTTL, "workload-cert-ttl", 19*time.Hour,
+	flags.StringVar(&naConfig.CAClientConfig.Org, "org", "", "Organization for the cert")
+	flags.DurationVar(&naConfig.CAClientConfig.RequestedCertTTL, "workload-cert-ttl", 19*time.Hour,
 		"The requested TTL for the workload")
-	flags.IntVar(&naConfig.RSAKeySize, "key-size", 2048, "Size of generated private key")
-	flags.StringVar(&naConfig.IstioCAAddress,
+	flags.IntVar(&naConfig.CAClientConfig.RSAKeySize, "key-size", 2048, "Size of generated private key")
```

Comment:
nit: define a `caClientConfig := &naConfig.CAClientConfig` at the beginning to save some typing?

Reply:
Yes, good idea :)